code/modules/mob/new_player/sprite_accessories/markings/eye_stuff.dm

```diff
@@ -52,3 +52,8 @@
 	name = "Third Eye"
 	icon_state = "thirdeye"
 	body_parts = list(BP_HEAD)
+
+/datum/sprite_accessory/marking/eye_stuff/closedeye
+	name = "Closed Eyes"
+	icon_state = "closedeyes"
+	body_parts = list(BP_HEAD)
```

Comment:
You need to name the state **in** the .dmi "closedeyes-head"
here in the .dm it stays "closedeyes"

Reply:
What I'm hearing is that I just need to give a little -head